File: tabbed/src/main/java/com/android/tabbed/dto/BudgetResponse.java
package com.android.tabbed.dto;

import java.time.YearMonth;

public class BudgetResponse {
    private Long id;
    private String userId;
    private YearMonth yearMonth;
    private Integer budget;
    private Integer spending;
    private Integer remaining;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public YearMonth getYearMonth() {
        return yearMonth;
    }

    public void setYearMonth(YearMonth yearMonth) {
        this.yearMonth = yearMonth;
    }

    public Integer getBudget() {
        return budget;
    }

    public void setBudget(Integer budget) {
        this.budget = budget;
    }

    public Integer getSpending() {
        return spending;
    }

    public void setSpending(Integer spending) {
        this.spending = spending;
    }

    public Integer getRemaining() {
        return remaining;
    }

    public void setRemaining(Integer remaining) {
        this.remaining = remaining;
    }
}

File: app/src/main/java/com/jth/chagokchagok/data/remote/dto/BudgetResponse.kt
package com.jth.chagokchagok.data.remote.dto

import java.time.YearMonth

data class BudgetResponse(
    val id: Long,
    val userId: String,
    val yearMonth: String,
    val budget: Int,
    val spending: Int,
    val remaining: Int
){
    fun getYearMonth(): java.time.YearMonth = java.time.YearMonth.parse(yearMonth)
}
